src/App.js

```diff
@@ -1,127 +1,102 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/jsx-no-target-blank */
+import React, { Component } from 'react'
+import './App.css'
 
 class App extends Component {
-  render() {
+  render () {
     return (
       <body>
-      <div id="container">
-
-        <!-- Calculator body -->
-        <div class="cal-body">
-
-          <!-- Header -->
-          <div class="header">
-            <h6>Electronic Calculator</h6>
-          </div>
-
-          <!-- Display -->
-
-          <div class="display">
-
-            <!-- Display: Memory Input -->
-
-            <div class="mem-input">
-              0
-            </div>
-
-            <!-- Display: Active Input -->
-
-            <div class="act-input">
-              0
+        <div id='container'>
```

Comment:
This is very similar but hard to see properly. The html comments and semicolons have been removed (for eslint) and  `class` has been changed to `className`. The important changes though are around the `=`and `0` button.